clients/js/src/cipher.ts

```diff
@@ -177,7 +177,7 @@ export class X25519DeoxysII extends Cipher {
     // Derive a shared secret using X25519 (followed by hashing to remove ECDH bias).
     const keyBytes = sha512_256.hmac
       .create('MRAE_Box_Deoxys-II-256-128')
-      .update(scalarMult(keypair.secretKey, peerPublicKey))
```

Comment:
Is this random refactoring, or actually needed to build cjs/esm?

Reply:
`nacl` isn't actually an ESM, but it is if you're using TS `esModuleInterop`. It won't work if you're not using TS.